docs/Common/Structs/index.md

```diff
@@ -1130,3 +1130,16 @@ There are no defined parameters for this struct
 |hybridAppPreference|[Common.HybridAppPreference](../enums/#hybridapppreference)|false||Specifies the user preference to use one specific app type or all available types|
 |endpoint|String|false|maxlength: 65535|If specified, which Core uses a client implementation of the connection type and attempts to connect to the endpoint when this app is selected (activated).<br>If omitted, Core won't attempt to connect as the app selection (activation) is managed outside of Core. Instead it uses a server implementation of the connection type and expects the app to connect|
 
+### WindowState
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|approximatePosition|Integer|true|minvalue: 0<br>maxvalue: 100|The approximate percentage that the window is open -<br>0 being fully closed, 100 being fully open|
+|deviation|Integer|true|minvalue: 0<br>maxvalue: 100|The percentage deviation of the approximatePosition.<br>e.g. If the approximatePosition is 50 and the deviation is 10,<br>then the window's location is somewhere between 40 and 60|
+
+### WindowStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|[Common.Grid](../structs/#grid)|true|||
+|state|[Common.WindowState](../structs/#windowstate)|true|||
```

Comment:
These links don't seem to work on the dev portal. Looking at the other structs, it seems we don't link to other structs within this page(See `grid` in `SeatLocation`, `seats` in `SeatLocationCapability`, etc.)

```suggestion
|location|Common.Grid|true|||
|state|Common.WindowState|true|||
```

Reply:
fixed in 3cea6e5